src/ORM/EagerLoadable.php

```diff
@@ -59,7 +59,7 @@ class EagerLoadable
      * A dotted separated string representing the path of associations
      * that should be followed to fetch this level.
      *
-     * @var string|null
+     * @var string
```

Comment:
Is this really true at runtime? After constructor is done?

Reply:
Yes, the constructor takes an array of config and that's used to set various properties one of which is this.